webpack-aliases.js

```diff
@@ -0,0 +1,60 @@
+const path = require('path');
+
+// This file should be auto-generated
+export default {
```

Comment:
The way you are referencing the button package [here](https://github.cerner.com/terra/cerner-consumer-theme/pull/8/files#diff-c952e4bc40cbd3318d484d1ef45b4108R4), I would think you only need to alias the repos.

Reply:
After looking at the webpack config documentation, an options could be to generate an index.js file for terra-core which would map to each package?

Reply:
I've moved this file into the theme-repo. Ran into some file path issues in how `__dirname` was being resolved.